microlith/prometheus/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -164,3 +164,37 @@ groups:
         description: Node {{ $labels.instance}} has {{ humanize $value }} open Data Service connections out of 60,000.
           Applications may fail to connect.
         remediation: Ensure your applications are appropriately closing their Couchbase connections.
+
+    - alert: CB90087-activeVbucketResidentRatio-warn
+      expr: kv_vb_perc_mem_resident_ratio{state="active"} < 0.1
+      for: 1m
+      labels:
+        job: couchbase_prometheus
+        kind: bucket
+        health_check_id: CB90087
+        health_check_name: activeVbucketResidentRatio
+        cluster: '{{ $labels.cluster }}'
+        node: '{{ $labels.instance }}'
+        bucket: '{{ $labels.bucket }}'
+        severity: warning
+      annotations:
+        title: " Resident Ratio of Active vBuckets is low "
```

Comment:
Trailing whitespace before quotes.

Reply:
%s/low /low./

Reply:
full stops at the end of title, description, remediation